packages/common/src/types.ts

```diff
@@ -46,7 +49,7 @@ export interface GenesisBlockConfig {
   excessDataGas?: string
 }
 
-export interface HardforkConfig {
+export interface HardforkTransitionConfig {
   name: Hardfork | string
   block: number | null // null is used for hardforks that should not be applied -- since `undefined` isn't a valid value in JSON
```

Comment:
Could we get rid of `null` now since our hardforks are stored in JS and not JSON files?  Could be a good future PR.